linter/orb.yaml

```diff
@@ -80,8 +85,19 @@ jobs:
         description: >
           Pre-commit package version.
         type: string
+      pre_commit_args:
+        default: --all-files --show-diff-on-failure
+        description: >
+          Pre-commit parameters
+        type: string
+      setup:
```

Comment:
This should be unnecessary: CircleCI already supports [pre-steps](https://circleci.com/docs/configuration-reference/#pre-steps-and-post-steps) for configuring this in a more generalized way. Is there a use-case that does not support?

Reply:
I didn't knew about `pre-steps`, which seems like a good way to do this. I'll remove that commit from the PR.